src/test/javascript/spec/component/plagiarism/plagiarism-sidebar.component.spec.ts

```diff
@@ -81,4 +82,15 @@ describe('Plagiarism Sidebar Component', () => {
 
         expect(comp.currentPage).toEqual(3);
     });
+
+    it('should handle changes correctly', () => {
+        comp.comparisons = [];
```

Comment:
Not quite sure about this one. Why do you define comparisons? How does this then relate to the change and pagedComparisons?

I suggest anyway not to test with an empty array but a filled one. Two items should be enough.

Reply:
@julian-christl Good catch! I must have forgotten to add some comparisons. I added some values now that make more sense